wiki/en/misc/1-index.html

```diff
@@ -30,26 +30,25 @@ <h2>What is Jamulus?</h2>
 </div>
 <div class="fx-row fx-row-center-xs">
   <div class="fx-col-100-xs fx-col-50-l">
-     <h2>Jamulus worldwide</h2>
-    All over the world Jamulus allows huge choirs to rehearse, loud rock bands to play, and brings folk and classical musicians together. It's being used for remote music lessons,
-    in schools and universities, in private and in public. Even just for the spoken word – all in real time on the Internet, as if you were there in person.
-    <h2>Help needed?</h2>
+     <h2>Worldwide</h2>
+    Huge choirs, rock bands, folk and classical musicians, remote music lessons, in schools and universities,
+    private or public. Even the spoken word — as if you were there in person.
+    <h2>Help?</h2>
     <p>
-      Have a look at the <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">documentation</a>. Especially consider the <a href="wiki/Client-Troubleshooting"
-        target="_blank" rel="noreferrer">troubleshooting section</a>!
+      Check out the <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">documentation</a>. Especially the <a href="wiki/Client-Troubleshooting"
+        target="_blank" rel="noreferrer">troubleshooting</a> section.
       You can also ask on the <a href="https://github.com/jamulussoftware/jamulus/discussions" target="_blank" rel="noreferrer">forums</a>.
     </p>
   </div>
   <div class="fx-col-100-xs fx-col-50-l">
-    <h2>Want to get involved?</h2>
+    <h2>Get involved</h2>
     <p>
-    Ideas? Found a bug? Want to contribute some code or help translating into your language? Since Jamulus is <a href="https://www.gnu.org/philosophy/free-sw.en.html" target="_blank" rel="noreferrer" title="What is free software?">free and open source software</a> (FOSS) licensed under the <a href="https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html" title="GNU General Public License, version 2" target="_blank" rel="noreferrer">GPL</a>, you can help us!<br/>
-    Take a look at our <a href="wiki/Contribution">contribution guidelines</a> to find out how. Everybody is welcome!
+    Use, see, change, and share — with all. <a href="https://github.com/jamulussoftware">write code</a> or <a href="https://hosted.weblate.org/engage/jamulus/">translate it on Weblate</a>.<br/>
```

Comment:
```suggestion
    Use, see, change, and share — with all. <a href="https://github.com/jamulussoftware">write code</a> or <a href="https://hosted.weblate.org/engage/jamulus/">translate it on Weblate</a>,<br/>
```

Reply:
I'm a native speaker of English, and this copy sound weirdly Shakespearean with all.

Reply:
The emphasis on Weblate is a bit forced. An icon on the bottom would be nicer.